Planomatic/DeliverablesPage.xaml

```diff
@@ -52,12 +52,38 @@
 
         </StackPanel>
 
-        <DataGrid ItemsSource="{Binding AllItemsView}" AutoGenerateColumns="False" CanUserAddRows="False" x:Name="AllDeliverableList" CanUserSortColumns="False">
+        <DataGrid ItemsSource="{Binding AllItemsView}" AutoGenerateColumns="False"
+                  CanUserAddRows="False" x:Name="AllDeliverableList" CanUserSortColumns="False"
+                  AlternationCount ="2" RowBackground="LightCyan" HorizontalScrollBarVisibility="Visible">

```

Comment:
Should use a theme color here, like this: {DynamicResource MaterialDesignToolBarBackground}, that looks pretty good here.

I also noticed that with your change only the alternate (cyan in your code) rows have a highlight when you mouse over.  We could probably remove the hover highlight entirely, or fix so it always works.

Reply:
Fixed the alternate rows so that both row types are being highlighted.
I prefer to set the style myself - feel free to change :-)